src/IntercomChannel.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace NotificationChannels\Intercom;
+
+use Intercom\IntercomClient;
+use GuzzleHttp\Exception\GuzzleException;
+use Illuminate\Notifications\Notification;
+use GuzzleHttp\Exception\BadResponseException;
+use NotificationChannels\Intercom\Exceptions\RequestException;
+use NotificationChannels\Intercom\Exceptions\MessageIsNotCompleteException;
+
+/**
+ * Class IntercomNotificationChannel.
+ */
+class IntercomChannel
+{
+    /**
+     * @var IntercomClient
+     */
+    private $client;
+
+    /**
+     * @param IntercomClient $client
+     */
+    public function __construct(IntercomClient $client)
+    {
+        $this->client = $client;
+    }
+
+    /**
+     * Send the given notification via Intercom API.
+     *
+     * @param mixed        $notifiable
+     * @param Notification $notification
+     *
+     * @return void
+     *
+     * @throws MessageIsNotCompleteException When message is not filled correctly
+     * @throws GuzzleException               Other Guzzle uncatched exceptions
+     * @throws RequestException              When server responses with a bad HTTP
+     *                                                                                         code
```

Comment:
code is floating here, assume it was meant to be on the line above :)

Reply:
Lol, doesn't mensioned this